app/views/histories/10_downing_street.html.erb

```diff
@@ -0,0 +1,307 @@
+<header class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <%= render "govuk_publishing_components/components/title", {
+      context: {
+        text: "History",
+        href: "/government/history",
+      },
+      title: "10 Downing Street",
+    } %>
+  </div>
+</header>
+
+<div class="govuk-grid-row">
+  <nav class="govuk-grid-column-one-third">
+    <span class="aspect-ratio-3:2">
+      <%= image_tag "history/buildings/number-10-300.jpg", alt: "The front door of 10 Downing Street", loading: "lazy" %>
+    </span>
+
+    <%= render "govuk_publishing_components/components/heading", {
+      text: "Contents",
+      font_size: 24,
+      padding: true,
+    } %>
+
+    <ol class="govuk-list govuk-list--bullet">
```

Comment:
This would be a visual change, but we do have a contents list component: https://components.publishing.service.gov.uk/component-guide/contents_list

Reply:
I'm all in favour of making these pages conform more to standard gov.uk styling